server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/InternalRange.java

```diff
@@ -157,6 +157,7 @@ private static String generateKey(double from, double to, DocValueFormat format)
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
+            // NOTE: we would actually need to use Version.V_6_4_0 which is not available, so we use the earliest possible version
```

Comment:
I don't think 7_17_1 is the earliest possible version though, right?

Reply:
Yes I will remove the comment...I had to change that in one of the following commits...my assumption about the earlier version was incorrect.